File: 140/Easy.clj

(require '[clojure.string :as str])

(defn variable-notation []
	(let [convention (read-line)]
		(let [input (read-line)]
			(if (= convention "2")
				(prn (str/upper-case (str/replace input #" " "_" )))
			(if (= convention "1")
				(prn (str/replace input #" " "_"))
			(if (= convention "0")
				(prn (str (first (str/split input #" ")) (str/join "" (map str/capitalize (rest (str/split input #" "))))))))))))

(variable-notation)

File: 144/Easy.py
n = int(raw_input())
d = {x[0]: int(x[1]) for i in xrange(n) for x in [raw_input().split()] }
d = {x[0]: d[x[0]]-int(x[1]) for i in xrange(n) for x in [raw_input().split()] }
print "\n".join(k+" "+str(v) for k,v in d.items() if v != 0)
